index.html

```diff
@@ -4,35 +4,50 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
     <link rel="stylesheet" href="style.css" />
     <title>Mo`s Portfolio</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com"> 
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
-<link href="https://fonts.googleapis.com/css2?family=Josefin+Sans&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
   </head>
-  <body>
-    <div class="hero">
-
-      <nav>
-          <h2 class="Logo">Portfolio</h2>
-         <a class="active" href="Home">Home</a>
-          <ul>
-              <li><a href="#">About Me</a></li>
-              <li><a href="#">Service</a></li>
-              <li><a href="#">Skills</a></li>
-              <li><a href="#">Contact</a></li>
-          </ul>
-          <img src="images/moon.png" alt="moon" id="icon">
-        </nav>
-    </div>
-
-
-
-
-</p>
-
-
-  </body>
+  <main>
+    <body>
+      <nav class="hero">
+        <h1 class="logo">Portfolio</h1>
+        <a class="active" href="Home">Home</a>
+        <ul>
+          <li><a href="#">About Me</a></li>
+          <li><a href="#">Service</a></li>
+          <li><a href="#">Skills</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <h2>About Me</h2>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa dolorem
+        officia aut natus temporibus aliquid modi voluptates odit perferendis
+        blanditiis iste qui maxime possimus ipsam soluta velit, at, recusandae
+        alias.
+      </p>
+
+      <h2>Contact</h2>
+
```

Comment:
here you can use a div class and put them into css without use flex 4 every logo/picture/link you post